msbuild/Xamarin.iOS.Tasks/Tasks/TaskItemFixer.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.iOS.Tasks
+{
+	public static class TaskItemFixer
+	{
+		const string XIFrameworkRootDir = "/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/";
```

Comment:
This won't be correct for .NET 6...

Reply:
This is only for the legacy SDK, we'll need to come up with a better name for it. The SDK repo will be the single source for the msbuild targets/tasks so we need this besides it's not net6 specific.